pocket_coffea/utils/rucio.py

```diff
@@ -191,11 +192,19 @@ def get_dataset_files_replicas(
                     )
                     outsite.append(site)
                     found = True
+                    
+            if not found and use_infn:
+                # The file was not found at any of the allowed sites
+                # But with this option we add the INFN redirector prefix
+                if len(list(rses.keys())) != 0:
```

Comment:
we are missing an `else` here, raising an exception about a dataset with completely no sites available

Reply:
I think it was not necessary. If `found=False` at the end, the message would be displayed later.
Also in the updated version, if the `found` flag not set, the exception will rise at the end of the function